vspace/src/main/webapp/WEB-INF/views/staff/modules/module.jsp

```diff
@@ -7,15 +7,64 @@
 
 <script>
 //# sourceURL=click.js
-$(document).ready(function($) {
+var slideIdToDelete = 0;
+//------------Deleting Slides-------------------
+function deleteSlide(slideId) {
+		$.ajax({
+	        url: "<c:url value="/staff/module/${module.id}/slide/" />" + slideId +'?${_csrf.parameterName}=${_csrf.token}',
+	        type: 'DELETE',
+	        cache       : false,
+	        contentType : false,
+	        success: function(data) {
+	        	$("#"+slideId).closest('.slide').remove();
+	        },
+	        error: function(data) {
+	        	var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>We are sorry but something went wrong. Please try again later.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+	        	$('.error').append(alert);
+	        }
+	    });
+}
+
+
+function checkSlideInSequence(slideId) {
+
+		$.ajax({
+	        url: "<c:url value="/staff/module/${module.id}/slide/" />" + slideId + "/sequences" + '?${_csrf.parameterName}=${_csrf.token}',
+	        type: 'GET',
+	        cache       : false,
+	        contentType : false,
+	        success: function(data) {
+	        	if(data.length > 0){
+	        		$("#deleteSlideAlert").show();
+		        	slideIdToDelete = slideId;
+	        	}
+	        	else{
+	        		deleteSlide(slideId);
+	        	}	
+	        }
+	    });
```

Comment:
same here

Reply:
I really need you to fix the indentation. why is the first line (and all the following) of the function indented by like 12 spaces or something? same thing above.